### PR TITLE
Contact form added in this milestone.

### PR DESCRIPTION
**Pull request by Mahdi-Niazi**
In this milestone I:

- Add Contact form to index.html form
- design the contact form based on figma template
- Add CSS property to the style.css file for contact form.

### CODE REVIEW

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @Mahdi-Niazi   👋

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request ❌
This is a pair-programming project and I only see in the commit history only one contributor 😨, Kindly collaborate with your coding partner and submit again once you are done 🙏  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Kindly create  a new PR that follows the Gitflow approach

Cheers, and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -300,6 +300,24 @@ <h2>About me</h2>
...
+                about new projects, 
+                so if you'd like to 
+                chat please get in
+                touch.
+            </h2>
+        </div>
+        <form action="https://formspree.io/f/xeqwblrq" method="POST" class="contact_form">
+            <input type="text" name="Name" id="Name" placeholder="First Name" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="Email" required>
+            <textarea name="message" id="message" rows="10" cols="20" maxlength="500" placeholder="Write me something..."></textarea>
```
Comment (abdulhamiid) [inline index.html]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), kindly ensure that a user cannot submit the form if one or more of the fields is empty. 💪

File: index.html [hunk 2]
```diff
@@ -300,6 +300,24 @@ <h2>About me</h2>
...
+            <button type="submit">Get in touch</button>
+        </form>
```
Comment (abdulhamiid) [inline index.html]: According to the project requirements, it is important that we stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Kindly implement the button interactions such as hover, active (when the button is pressed), disabled, and enable (default state) styled as shown in the Figma template. 🤗🛠

image

File: index.html (hunk 1 above)
Comment (abdulhamiid) [inline index.html]: Figma
image

Your design
image

I would suggest that we reduce the Form margin (right & left) to stick to the design on Figma as much as possible. 🤗 part of our learning objective is to understand how to parse a Figma design to create a UI. 💪

File: index.html [hunk 3]
```diff
@@ -300,6 +300,24 @@ <h2>About me</h2>
...
+    </section>
 </body>
```
Comment (abdulhamiid) [inline index.html]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), the section you need to build for this milestone is:
- the contact form:
- and the footer (if present in the design)

Kindly add the Footer, following the template that you chose. 🤓🛠

Figma
image

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi Team 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

[REQUIRED] - I'm unable to add an inline comment to your readme, please check the requested change below

image

image

Kindly fix the broken link to your LinkedIn profile, let's avoid broken links in our readme for a professional repository. 🤗👍

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (stanleySimeon) [reply index.html]: As mentioned by the previous reviewer kindly, I suggest that we reduce the Form margin (right & left) to stick to the design on Figma as much as possible. 🤗 part of our learning objective is to understand how to parse a Figma design to create a UI. 💪

> ## The Figma Design
>
>Screenshot 2023-06-06 at 12 02 45 PM
>
> ## Your design
>
>Screenshot 2023-06-06 at 11 54 59 AM

File: index.html (hunk 3 above)
Comment (stanleySimeon) [reply index.html]: Great work on implementing the changes required by the previous reviewer, but there is still something you'll need to fix to go through the next step, kindly I suggest you stick to the design as much as possible `(e.g., font, colors, images, text, margins)` using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma Design
>
>Screenshot 2023-06-06 at 12 11 02 PM
>
> ## Your Design
>
>Screenshot 2023-06-06 at 11 57 37 AM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @Mahdi-Niazi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Mahdi-Niazi, @s2jack, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Reduced the Form margin (right & left) to stick to the design on Figma as much as possible. ✔️ 
- All linters are passing :heavy_check_mark: 
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
- [ ]  Next time, and for future commits, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Bad Commit Messages :x:
image

### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**